includes/wikia/cache/TitleBatch.php

```diff
@@ -218,17 +218,18 @@ static public function newFromIds( $ids, $dbType ) {
 	 * @param $conds array|string Conditions
 	 * @param $fname string Function name
 	 * @param $options array Query options
+	 * @param $joinConditions array Join conditions
 	 * @param $dbType int Database connection type (doesn't support DB_SLAVE_BEFORE_MASTER)
 	 * @return TitleBatch titleBatch contains existing articles
 	 */
 	static public function newFromConds( $tables, $conds, $fname = 'TitleBatch::newFromConds',
```

Comment:
changes worth communicating wider - maybe other teams could update some of their code as well?